metricbeat/mb/lightmodules_test.go

```diff
@@ -195,18 +195,22 @@ func TestNewModuleFromConfig(t *testing.T) {
 		"normal module": {
 			config:         common.MapStr{"module": "foo", "metricsets": []string{"bar"}},
 			expectedOption: "default",
+			expectedQuery:  nil,
```

Comment:
In the original PR to `master`, the value here is `QueryParams{}` instead of `nil`. Shouldn't it be the same here?

Reply:
Ah, never mind. I see that you are backporting 2 PRs in one here.

Reply:
Would you mind doing the backports separately? It'd make it trivial to review them, especially given the size of this PR here.